versioned_docs/version-2.0/tutorials/Administrators/manage_spend.mdx

```diff
@@ -0,0 +1,157 @@
+---
+sidebar_position: 1
+---
+
+import {
+  CodeTabs,
+  python,
+  typescript,
+  shell,
+} from "@site/src/components/InstructionsWithCode";
+
+# Optimize tracing spend on LangSmith
+
+This tutorial walks through optimizing your spend on LangSmith. In it, we will learn how to optimize existing spend
+and prevent future overspend on a realistic real-world example. 
+
+## Tutorial background
+In this tutorial, we will use an existing LangSmith tenant with high usage. Concepts can be transferred to your own tenant. 
+
+## Understand your current usage
+The first step of any optimization process is to understand current usage. LangSmith gives two ways to do this: Usage Graph 
+and Invoices.
+
+### Usage Graph
+The usage graph lets us examine how much of each usage based pricing metric we have consumed lately. It does not give insight
+into spend. 
+
+We can navigate to the Usage Graph under `Settings` -> `Usage and Billing` -> `Usage Graph`. 
+
+![](./static/p1usagegraph.png)
+
+We see in the graph above that there are two usage metrics that LangSmith charges for, LangSmith Traces (Base Charge) and 
+LangSmith Traces (Extended Data Retention Upgrades). The first metric tracks all traces that you send to LangSmith. The second
+tracks all traces that have our Extended 400 Day Data Retention. For more details, see [TODO: LINK]
```

Comment:
Where do we think we should put the docs on how auto upgrades work? It feels almost like the pricing docs need to expand to usage/pricing, and multiple sections now. data retention based pricing, auto upgrades, usage limits, etc all need explanation in some places